pkg/process/net/check_windows.go

```diff
@@ -0,0 +1,58 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+//go:build windows
+
+package net
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+
+	sysconfig "github.com/DataDog/datadog-agent/cmd/system-probe/config"
+	"github.com/DataDog/datadog-agent/pkg/collector/corechecks/system/wincrashdetect/probe"
+	"github.com/DataDog/datadog-agent/pkg/util/log"
+)
+
+const (
+	checksURL = "http://%s/%s/check"
+)
+
+// GetCheck returns the check output of the specified module
+func (r *RemoteSysProbeUtil) GetCheck(module sysconfig.ModuleName) (interface{}, error) {
+	url := fmt.Sprintf(checksURL, r.path, module)
```

Comment:
Should we bail early if `module != sysconfig.WindowsCrashDetectModule`?

Reply:
this is taken from the equivalent in `check.go`.  

we could do an early return, but adding additional remote checks (as in linux) would result in having a big `if/else` both before and after the request.

I'd be happy to change it if you think it's necessary @hmahmood

Reply:
Running the check only to discard it later doesn't seem right. @xornivore could you shed some light on this please?

Reply:
fair enough.  I'll change it but note the linux side then should have the same change.

Reply:
Linux does already return an error if you use an invalid check name